src/TenPinBowling.cpp

```diff
@@ -93,7 +93,7 @@ std::string ExtractPointsFromString(std::string const & playerFrames)
     std::string str = playerFrames;
     auto const pos=playerFrames.find_last_of(':');
     if(pos != std::string::npos)
-        str.erase(str.begin(), str.begin()+pos);
+        str.erase(str.begin(), str.begin()+pos+1);
```

Comment:
A nie powinno być tak:
`str.erase(str.begin(), pos+1);`
pos już jest iteratorem, nie musisz dodawać go do begin().
Stąd chyba też UT-ki nie przechodzą :)

Reply:
Przeszło, ale nie rozwiązałem tego problemu tylko zamiotłem go pod dywan. A co do pos to .find_last_of() zwraca size_type i dlatego mi to kompilator nie akceptował.